src/show/tasks.py

```diff
@@ -0,0 +1,18 @@
+from celery import shared_task
+from lst.models import Lst
+from show.models import Show
+
+
+@shared_task
+def add_show_to_lists(show_id, list_ids, user):
```

Comment:
are you sure you can pass in a user? have you tested this yet? the last time i tried to pass in a django object into a task, it didn't work, so i had to pass in `user_id`...

Reply:
I tested and It's been working for me :O

Reply:
because it's not an async task 😭 you need to use `add_show_to_lists.delay()`

Reply:
I see why its because delay wasn't called . now changed to user_id